cmd/upgrade/roles/cmd.go

```diff
@@ -927,6 +939,7 @@ func buildOperatorRoleCommandsFromCluster(
 		upgradePoliciesCommands := awscbRoles.ManualCommandsForUpgradeOperatorRolePolicy(
 			awscbRoles.ManualCommandsForUpgradeOperatorRolePolicyInput{
 				HasPolicy:                hasPolicy,
+				PolicyAttached:           policyAttached,
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢